lang/python/python-cryptography/Makefile

```diff
@@ -20,6 +20,10 @@ PKG_LICENSE_FILES:=LICENSE.APACHE LICENSE.BSD
 PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>, Alexandru Ardelean <ardeleanalex@gmail.com>
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-cryptography-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=libffi/host
+
+HOST_PYTHON_PACKAGE_BUILD_DEPENDS:="cffi>=1.8,!=1.11.3"
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:="cffi>=1.8,!=1.11.3"
```

Comment:
makes sense;
to be honest, i don't remember quite well; but i think there may have been a time when cffi needed cffi on the host side (at some older version);
this whole `HOST_PYTHON[3]_PACKAGE_BUILD_DEPENDS` mechanism came to exist as a need to support to add `python3-cffi`; python-cffi already existed for some time